Orm/Xtensive.Orm/Orm/Providers/SqlCompiler.Index.cs

```diff
@@ -7,16 +7,38 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Expressions;
+using Xtensive.Core;
+using Xtensive.Linq;
+using Xtensive.Orm.Internals;
 using Xtensive.Orm.Model;
 using Xtensive.Orm.Rse.Providers;
+using Xtensive.Reflection;
 using Xtensive.Sql;
 using Xtensive.Sql.Dml;
 using IndexInfo = Xtensive.Orm.Model.IndexInfo;
 
 namespace Xtensive.Orm.Providers
 {
-  partial class SqlCompiler 
+  public partial class SqlCompiler
```

Comment:
Do we really need to expose it `public`?

Reply:
It was public. Take a look at the main declaration in ```Orm/Xtensive.Orm/Orm/Providers/SqlComliler```.

https://github.com/DataObjects-NET/dataobjects-net/blob/3263519079d7f59496b9b7649c03ade88a949ef7/Orm/Xtensive.Orm/Orm/Providers/SqlCompiler.cs#L22-L26

 I just made this part declaration clearer.